packages/web-components/src/components/content-item/content-item-copy.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
I know this is an internal component but wondering if we should start to document the things in here regardless if we're importing into a README or not, it would at least help other developers when maintaining the code.

Reply:
@jeffchew Good question - [Web Component Analyzer](https://github.com/runem/web-component-analyzer#readme) automatically harvests the API from ancestor class, and thus the API doc in README reflects ancestor class's API. I think it suffices.